tests/utils/stl/util.py

```diff
@@ -0,0 +1,293 @@
+#===----------------------------------------------------------------------===##
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+#===----------------------------------------------------------------------===##
+
+from contextlib import contextmanager
+import errno
+import os
+import platform
+import signal
+import subprocess
+import sys
+import tempfile
+import threading
+
+
+# FIXME: Most of these functions are cribbed from LIT

```

Comment:
These files should use TODO or FIXME consistently. (Or TRANSITION, the STL's preferred form.)

Reply:
I use TODO, the llvm devs seemed to switch between FIXME and TODO, I'll transition all the relevant ones for us to TRANSITION.